### PR TITLE
#36: Moved assembly/all-dependencies.xml to src/assembly/all-dependencies.xml

### LINKED ISSUE #36: Move assembly/all-dependencies.xml
## Problem 

I noticed that this file is stored in the src directory for RLS.
For some other projects it's stored in the root directory.

We need to check it and agree on a place where we put this file

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #36

### CODE REVIEW

File: src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java [hunk 1]
```diff
@@ -36,4 +44,20 @@ void testFixConfiguration(final String removeXpath) throws ParserConfigurationEx
...
+        runAllFixesGeneratedByValidation(pom, Collections.emptyList());
+        assertThat(pom,
+                not(hasXPath("/project/build/plugins/plugin/configuration/systemPropertyVariables/test.coverage")));
+    }
+
+    @Test
+    void testFixWithUdfIntegration() throws ParserConfigurationException, SAXException, IOException {
+        final Document pom = readXmlFromResources(POM_WITH_NO_PLUGINS);
+        runAllFixesGeneratedByValidation(pom, List.of(ProjectKeeperModule.UDF_COVERAGE));
+        assertThat(pom, EvaluateXPathMatcher.hasXPath(
```
Comment (morazow) [inline src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java:60]:
```suggestion
        assertThat(pom, hasXPath(
```

Comment (morazow) [reply src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java:60]: To make it consistent with above test.

**Review by morazow — APPROVED** [review]

File: src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java (hunk 1 above)
Comment (jakobbraun) [reply src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java:60]: It's another method with two parameters

Comment (morazow) [reply src/test/java/com/exasol/projectkeeper/validators/pom/plugin/FailsafePluginPomValidatorTest.java:60]: Ahh, I see :+1:

**Pull request merged**